frontend/js/pages/BattleDetails.js

```diff
@@ -1,4 +1,6 @@
+import get from 'lodash/get';
 import isEmpty from 'lodash/isEmpty';
```

Comment:
Same destructuring import matter here.
```suggestion
import { get, isEmpty } from 'lodash';
```